xonsh/ptk/key_bindings.py

```diff
@@ -107,6 +107,15 @@ def __call__(self, cli):
         return bool(at_end and not last_line)
 
 
+class ShouldConfirmCompletion(Filter):
+    """
+    Check if completion needs confirmation
+    """
+    def __call__(self, cli):
+        return (builtins.__xonsh_env__.get('COMPLETION_CONFIRM', False)
```

Comment:
You don't need to provide False here, since it is in the `Env` defaults already.  In general, we try to define the default only in one place.